.github/workflows/publish.yml

```
@@ -0,0 +1,78 @@
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        description: The environment to deploy.
+
+jobs:
+  test:
+    call-shared:
+      uses: ./.github/workflows/shared-workflow.yml
+
+  detect-environments:
```

actionlint (1.7.7)

job "detect-environments" needs job "lint" which does not exist in this workflow

(job-needs)